go/vt/vttablet/tabletmanager/vreplication/flags.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2022 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vreplication
+
+import (
+	"time"
+
+	"github.com/spf13/pflag"
+
+	"vitess.io/vitess/go/vt/servenv"
+)
+
+var (
+	retryDelay          = 5 * time.Second
+	maxTimeToRetryError time.Duration // default behavior is to keep retrying, for backward compatibility
+
+	tabletTypesStr = "in_order:REPLICA,PRIMARY"
+
+	relayLogMaxSize  = 250000
+	relayLogMaxItems = 5000
+
+	copyPhaseDuration   = 1 * time.Hour
+	replicaLagTolerance = 1 * time.Minute
+
+	vreplicationHeartbeatUpdateInterval = 1
+	vreplicationExperimentalFlags       = int64(0x01) // enable vreplicationExperimentalFlagOptimizeInserts by default
+	vreplicationStoreCompressedGTID     = false
+)
+
+func registerVReplicationFlags(fs *pflag.FlagSet) {
+	fs.DurationVar(&retryDelay, "vreplication_retry_delay", retryDelay, "delay before retrying a failed workflow event in the replication phase")
+	fs.DurationVar(&maxTimeToRetryError, "vreplication_max_time_to_retry_on_error", maxTimeToRetryError, "stop automatically retrying when we've had consecutive failures with the same error for this long after the first occurrence")
+
+	// this are the default tablet_types that will be used by the tablet picker to find source tablets for a vreplication stream
+	// it can be overridden by passing a different list to the MoveTables or Reshard commands
+	fs.StringVar(&tabletTypesStr, "vreplication_tablet_type", tabletTypesStr, "comma separated list of tablet types used as a source")
```

Comment:
have you looked at `fs.StringSliceVar` ? it might let us delete some extra code for doing the splitting

Reply:
This is not a simple list. We support options like `in_order:REPLICA,PRIMARY`